### DISCUSSION TITLE
Adding userstring before the clipped text while using html2tw macro

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#103] [Q&A] [answered]
I use html2tw macro by Mark in my wiki along with tiddlyclip. https://marxsal.github.io/various/playground.html

<img width="887" alt="image" src="https://user-images.githubusercontent.com/67494083/190874957-7388447d-2237-4daf-83b9-c676c47c8941.png">

<img width="876" alt="image" src="https://user-images.githubusercontent.com/67494083/190874971-d7469e16-567b-4df5-8773-a024605c8d73.png">

<img width="870" alt="image" src="https://user-images.githubusercontent.com/67494083/190874976-61570774-50ec-4b19-91bf-08f108a7c47d.png">

I want to add a userstring `before` the clipped text. How to do that ?

**Comment by buggyj** [comment]
I cannot see from these tiddlers what you are doing. 
Can you provide a link to the tiddlywiki? Or provide the source of the tiddlers including the tiddlyclip config.

**Reply by arunnbabu81** [reply]
https://demo-complete.tiddlyhost.com/

Here is the demo
Click on the Tiddlyclip demo link in the topbar to see the Tiddlyclipconfig used

The purpose is to add a Heading text using userstring above the text clipped using the html2tw macro (convertdet)

This will help in easy usage of Section Editor by kookma in minimal steps

![Screenshot_20220918-130622_Tiddloid](https://user-images.githubusercontent.com/67494083/190891148-3aba4f26-f36f-4f37-b3f8-0f11fbfc6462.jpg)

![Screenshot_20220918-130643_Tiddloid](https://user-images.githubusercontent.com/67494083/190891155-c44f3b71-0a17-4ea8-98d6-9d56bc98fb55.jpg)

**Comment by buggyj** [comment, marked as answer]
Maybe in tiddler 'ConvertDet'


`[ {"#newdata":"((*@userstring*))\n\n((*@html2tw(@web)*))\n\n---—\n\n"} ] `

and add "userstring"  to the Modes column in the tiddlyclip config

**Reply by arunnbabu81** [reply]
It works. Thank you @buggyj
